examples/api/test/material/input_decorator/input_decoration.suffix_icon_constraints.0_test.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/input_decorator/input_decoration.suffix_icon_constraints.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Shows all decorations', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.SuffixIconConstraintsExampleApp(),
+    );
+    expect(find.text('InputDecoration Sample'), findsOneWidget);
+
+    expect(find.byType(TextField), findsNWidgets(2));
+    expect(find.byIcon(Icons.search), findsNWidgets(2));
+    expect(find.text('Normal Icon Constraints'), findsOneWidget);
+    expect(find.text('Smaller Icon Constraints'), findsOneWidget);
+
+    final Finder normalIcon = find.descendant(
+      of: find.ancestor(
+        of: find.text('Normal Icon Constraints'),
+        matching: find.byType(TextField)
+      ),
+      matching: find.byIcon(Icons.search)
+    );
+    final Finder smallerIcon = find.descendant(
+      of: find.ancestor(
+        of: find.text('Smaller Icon Constraints'),
+        matching: find.byType(TextField)
+      ),
+      matching: find.byIcon(Icons.search)
+    );
+
+    expect(tester.getSize(normalIcon).longestSide,
+      greaterThan(tester.getSize(smallerIcon).longestSide));
+  });
+
+  testWidgets('Is suffix icon', (WidgetTester tester) async {
```

Comment:
```suggestion
  testWidgets('Is suffix icon', (WidgetTester tester) async {
```

Not sure what this means.

Reply:
I changed the description to `suffixIcons are placed right of hintText`. The goal of this text is to ensure the icon in the example is actually a suffix icon (after content) and not a prefix icon. 

While this is not the main point the example is trying to illustrate, I believe it is necessary to differentiate between the prefixIconConstraints and the suffixIconConstraints example.